lms/djangoapps/teams/static/teams/js/collections/team_membership.js

```diff
@@ -8,15 +8,22 @@
 
                     this.course_id = options.course_id;
                     this.username = options.username;
+                    this.privileged = options.privileged;
                     this.perPage = options.per_page || 10;
                     this.server_api['expand'] = 'team';
-                    this.server_api['course_id'] = function () { return encodeURIComponent(this.course_id); };
+                    this.server_api['course_id'] = function () { return encodeURIComponent(options.course_id); };
                     this.server_api['username'] = this.username;
                     delete this.server_api['sort_order']; // Sort order is not specified for the TeamMembership API
                     delete this.server_api['order_by']; // Order by is not specified for the TeamMembership API
                 },
 
-                model: TeamMembershipModel
+                model: TeamMembershipModel,
+
+                canUserCreateTeam: function() {
+                    // Note: non-privileged users can only be in one team, so they
```

Comment:
Please clarify comment to include the bit of information that non-privileged are automatically added to a team on creation, whereas privileged users are not. I do believe that all users (even privileged) will only be allowed to be in a single team. Thus on the Team page, the ability to join a team only needs to check team size and whether or not the user is already a member of a team (it does not need to check privileged status).